testsuite/elytron/pom.xml

```diff
@@ -20,10 +20,39 @@
         <jbossas.ts.dir>${jbossas.ts.integ.dir}/..</jbossas.ts.dir>
         <jbossas.project.dir>${jbossas.ts.dir}</jbossas.project.dir>
         <wildfly.home>${project.basedir}/target/wildfly-core</wildfly.home>
+        <version.javax.enterprise>1.2</version.javax.enterprise>
 
         <ts.elytron.cli>enable-elytron.cli</ts.elytron.cli>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.directory.server</groupId>
+                <artifactId>apacheds-parent</artifactId>
+                <version>${version.org.apache.ds}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+
+            <dependency>
+                <groupId>javax.enterprise</groupId>
```

Comment:
IMO this shouldn't be in the parent POM. It's used just to workaround the ApacheDS limitations by providing customizable "annotations" (`extends AnnotationLiteral<T> implements T`).